package.json

```diff
@@ -18,8 +18,12 @@
   },
   "homepage": "https://github.com/danutDiac/tsa-node-project#readme",
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
we dont need axios

Reply:
Fixed in 01d971c68cfed900c4806fea3b50bf2f2539f139